src/test/java/com/google/api/codegen/testdata/java_page_streaming_response_library.baseline

```diff
@@ -47,43 +47,282 @@ import javax.annotation.Generated;
 @ExperimentalApi
 public class PagedResponseWrappers {
 
-  public static class ListShelvesPagedResponse extends PagedListResponseImpl<ListShelvesRequest, ListShelvesResponse, Shelf> {
+  public static class ListShelvesPagedResponse implements PagedListResponse<Shelf> {
+
+    private final PagedListResponseContext<RequestT, ResponseT, ResourceT> context;
```

Comment:
I don't think you can parameterize a member variable with more parameters than are present in the type. I think you need the specific types here: `<ListShelvesRequest, ListShelvesResponse, Shelf>`.